index.bs

```diff
@@ -720,22 +720,24 @@ these steps.
    following steps:
 
    1. Let |transport| be [=this=].
-   1. If |transport|'s {{WebTransport/state}} is `"closed"` or `"failed"`,
-      immediately return a new [=rejected=] promise with a newly created
-      {{InvalidStateError}} and abort these steps.
+   1. If |transport|'s [=WebTransport/[[State]]=] is `"closed"` or `"failed"`,
+      return a new [=rejected=] promise with an {{InvalidStateError}}.
    1. Let |p| be a new promise.
-   1. Return |p| and continue the remaining steps [=in parallel=].
-   1. [=BidirectionalStream/create=] a {{BidirectionalStream}} with |transport|
-      and |p| when all of the following conditions are met:
-       1. The |transport|'s {{WebTransport/state}} is `"connected"`.
-       1. Stream creation flow control is not being violated by exceeding the
-          max stream limit set by the remote endpoint, as specified in
-          [[!QUIC]].
-       1. |p| has not been [=settled=].
-   1. Queue a task to [=reject=] |p| with a newly created {{InvalidStateError}}
-      when all of the following conditions are met:
-       1. The |transport|'s state is `"closed"` or `"failed"`.
-       1. |p| has not been [=settled=].
+   1. Run the following steps [=in parallel=], but abort them whenever |transport|'s
+      [=WebTransport/[[State]]=] becomes `"closed"` or `"failed"`, and instead
+      [=queue a network task=] with |transport| to [=reject=] |p| with an {{InvalidStateError}}.
```

Comment:
I like this phrasing. Can you update [createUnidirectionalStream](https://pr-preview.s3.amazonaws.com/w3c/webtransport/pull/280.html#dom-webtransport-createunidirectionalstream) to match?

Reply:
Has this been addressed in #279?